cmd/gitea.go

```diff
@@ -211,9 +202,9 @@ func generateGiteaArgoApp(profile *kubeflowv1.Profile, replicas int32) (*argocdv
 				Name: "in-cluster",
 			},
 			Source: argocdv1alph1.ApplicationSource{
-				RepoURL: "https://github.com/blairdrummond/gitea-helm.git",
-				TargetRevision: "fix-extra-volume-mounts",
-				Path: ".",
+				RepoURL: "https://github.com/StatCan/aaw-argocd-manifests.git",
+				TargetRevision: "aaw-dev-cc-00",
```

Comment:
Can \ should the revision be dynamic so it will work when sent to prod?

Reply:
@vexingly good question. We are planning on handling that when we implement the production deployment, as right now this controller is only intended for the dev cluster.

Reply:
You'll want to add this to the profiles-controller helm values and pass the jsonnet branch into the values.

Like here:

https://github.com/StatCan/aaw-argocd-manifests/blob/901800ef215a12ae0605a3d393c81c15b79f874c/daaas-system/profile-controllers/profiles-controller/application.jsonnet#L63-L64